api/delete-image.js

```
@@ -0,0 +1,66 @@
+        return res.status(405).json({ error: 'Method not allowed' });
+    }
+
+    try {
+        const { public_id } = req.body;
+
+        if (!public_id) {
+            return res.status(400).json({ error: 'Missing public_id' });
+        }
+
+        // Config
+        const cloudName = process.env.CLOUDINARY_CLOUD_NAME;
```

ESLint

[error] 'process' is not defined.

(no-undef)